src/console/web_sys.rs

```diff
@@ -1,79 +1,105 @@
+use paste::paste;
+
 #[doc(hidden)]
-pub extern crate web_sys;
+pub mod __macro {
+    pub use js_sys::Array;
+    pub use wasm_bindgen::JsValue;
+    pub use web_sys::console;
+}
 
-use paste::paste;
+#[doc(hidden)]
+#[macro_export]
+macro_rules! __jsvalue {
+    ($value: expr) => {
+        ::std::convert::Into::<$crate::__macro::JsValue>::into($value)
```

Comment:
Link to related conversation: https://discord.com/channels/701068342760570933/701068343431528490/755478012060893304